documentation/tutorials/github.md

```diff
@@ -137,3 +137,23 @@ defmodule MyApp.Accounts.User do
 
 end
 ```
+
+Ensure you set the `hashed_password` to `allow_nil?` if you are also using the password strategy.
```

Comment:
@zachdaniel It's actually under the Get Started Page though: https://ash-hq.org/docs/guides/ash_authentication/latest/tutorials/get-started#example

Reply:
Not sure I understand what you mean. The getting started guide has you using the password strategy.